fairseq/checkpoint_utils.py

```diff
@@ -135,6 +135,19 @@ def load_checkpoint(args, trainer):
     return extra_state, epoch_itr
 
 
+def load_checkpoint_tpu(args, trainers, device_preloaded):
```

Comment:
do you need `load_checkpoint_tpu` instead of just using `load_checkpoint` above?

Reply:
yes, this is needed in the multithreading context.